### PR TITLE
Added yaml file for correct opening of empty_world simulation

### PR DESCRIPTION
**Pull request by LucasJSch**
empty_world simulation wasn't working in previous commit. Fixed it.

### CODE REVIEW

**Comment by LucasJSch** [comment]
# Ready for review

File: navigation/ca_move_base/maps/empty.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+free_thresh: 0.196
+image: playground.pgm
```
Comment (eborghi10) [inline navigation/ca_move_base/maps/empty.yaml]: [This parameter](http://wiki.ros.org/map_server#YAML_format) should point to a `pgm` file that corresponds to an empty environment.

I recommend you to compare you approach with this one ([empty_world](https://github.com/utexas-bwi/segbot_rocon/tree/bc96568f23b917d80744c6730d3adb76b31b2f55/segbot_concert_services/maps/empty_world)) and also include the missing `pgm`.

**Review by eborghi10 — CHANGES_REQUESTED** [review]
One minor comment, PTAL.

**Comment by LucasJSch** [comment]
# Ready for review

**Review by eborghi10 — APPROVED** [review]

**Pull request merged**